types/twitch/index.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for Twitch 1.0
+// Project: https://dev.twitch.tv/docs/embed/video-and-clips/
+// Definitions by: Ignacio Degregori <https://github.com/idegre>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
```suggestion
```
pls remove this, no longer relevant in the ecosystem

Reply:
Ah, I didnt know. NP